installer/templates/phx_umbrella/config/extra_config.exs

```diff
@@ -1,3 +1,4 @@
+import Config
```

Comment:
I think this change is wrong. If we keep this, we will have two `import Config` on the file we are augmenting. What was the reason for this change? :)

Reply:
You're 100% correct. This was done without fully realizing the consequences.

My next task was figuring out why there were two `import Config` statements 😁. Thanks for pointing this out!